src/Common/src/Interop/Unix/libcurl/Interop.libcurl_curlcode.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
This is already in the libcurl folder, so the file name probably doesn't also need it.  The capitlization should also match the type name. e.g. ```Interop.CURLcode.cs```.